### PR TITLE
bug 1442828 adding identity provider planning

### PR DESCRIPTION
**Pull request by kalexand-rh**
@thom-at-redhat , @simo5, I'm trying to address https://bugzilla.redhat.com/show_bug.cgi?id=1442828. PTAL

### CODE REVIEW

**Comment by simo5** [comment]
SGTM

**Comment by kalexand-rh** [comment]
@jianlinliu, PTAL

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

File: install_config/install/planning.adoc [hunk 1]
```diff
@@ -43,7 +43,13 @@ example as a basis for your environment.
  containerized]?_ Both installations provide a working {product-title}
  environment, but you might have a preference for a particular method of
  installing, managing, and updating your services.
-
+ 
+ * _Which identity provider do you use for 
+ xref:../../install_config/configuring_authentication.adoc#install-config-configuring-authentication[authentication]?_
+ If you already use a supported identity provider, it is a best practice to 
+ configure {product-title} to use it during installation. 
+
```
Comment (mburke5678) [inline install_config/install/planning.adoc]: @kalexand-rh I am probably being too picky here. I first read  _configure {product-title} to use it during installation_ to mean _use the identity provider for the installation_. Maybe  something like:
s/configure {product-title} to use it during installation/configure that provider before installing {product-title}.

**Comment by mburke5678** [comment]
@kalexand-rh One nit. Otheerwise LGTM

File: install_config/install/planning.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install_config/install/planning.adoc]: @mburke5678, if it's not coming across well, then it should change. It's best to add the identity provider info to the inventory file during installation as opposed to going back after installation to mod the configuration. You're configuring {product-title}, not your identity provider.

does this sound better?
s/configure {product-title} to use it during installation./configure {product-title} to use that identity provider during xref:../../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installation].

Comment (mburke5678) [reply install_config/install/planning.adoc]: Sounds good to me!

**Comment by kalexand-rh** [comment]
@stuartchuan, will you PTAL?

**Comment by stuartchuan** [comment]
@kalexand-rh The changes LGTM

**Comment by kalexand-rh** [comment]
Thanks! Merging.

**Pull request merged**